tg_bot.py

```diff
@@ -40,9 +41,15 @@ def display_tweet(self, context):
         print('starting display parameter')
         job = context.job
         global KILL_SIGNAL
+        previous_date =  datetime.now() + timedelta(days=-1)
```

Comment:
Doesn't datetime.now() - timedelta(1) looks better?

Reply:
yes why not :)

Reply:
Please update then :)